features/step_definitions/networking.rb

```diff
@@ -631,3 +631,13 @@
     raise "Failed to delete the docker container." unless @result[:success]
   }
 end
+
+
+Given /^the multus is enabled on the cluster$/ do
+  ensure_admin_tagged
+
+  desiredMultusReplicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:desired]
```

Comment:
@bmeng looks good, one minor issue is that it looks like you are ready to jump into Golang already :), but for ruby the naming convention for variables are snake case (https://github.com/rubocop-hq/ruby-style-guide#snake-case-symbols-methods-vars)

Reply:
Aha, yeah... read golang code more recently.

Have updated the names. Thanks.